packages/bpk-component-carousel/src/utils.test.tsx

```diff
@@ -0,0 +1,18 @@
+import { renderHook } from '@testing-library/react-hooks';
+
+import { useScrollToInitialImage } from './utils';
+
+describe('useScrollToInitialImage', () => {
+  it('should scroll to initial image on mount', () => {
```

Comment:
this test will still pass without the code optimisation right, how do we test that the optimisation works?